assets/css/style.css

```diff
@@ -0,0 +1,150 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700&display=swap");
+
+* {
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  box-sizing: border-box;
+  padding: 0;
+  margin: 0;
+}
+
+body {
+  background-color: #e5e5e5;
+}
+
+.container-generic {
+  max-width: 75%;
+  margin: 0 auto;
+}
+
+.hero {
+  padding: 0 0 30px 0;
+  background-color: #fff;
+  border-radius: 0 0 0 4rem;
+}
+
+.nav-container {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  max-width: 82%;
+  margin: 0 auto;
+}
+
+.menu-list {
+  list-style: none;
+  color: white;
+}
+
+.menu-item {
+  display: inline-block;
+  padding: 25px;
+  font-size: 15px;
+  font-weight: 500;
+}
+
+a {
+  text-decoration: none;
+  color: #344563;
+}
+
+.nav-logo {
+  display: flex;
+  font-weight: 700;
+  font-size: 18px;
+  color: #6070ff;
+}
+
+.mob-menu {
+  padding: 25px;
+  padding-right: 0;
+}
+
+.mob-menu-bar {
+  width: 20px;
+  height: 3px;
+  background-color: #344563;
+  margin-top: 4px;
+}
+
+.bio-content {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  height: auto;
+  width: 100%;
+  margin: 0 auto;
+  background-image: url(/assets/img/header-mob.png);
```

Comment:
- [ ] As the previous reviewer suggested, kindly try as much as possible to stick to the Figma template design ☺️. Please make sure to verify the [image file path](https://www.w3schools.com/html/html_filepaths.asp). This will help you to make your portifolio professional and also matching the Figma design ☺️.

Your design 😎 
![image](https://user-images.githubusercontent.com/99394374/235327019-0446ff8c-a9e9-4937-8aa6-2906ba0b01cb.png)